docs/_data/shapes.csv

```diff
@@ -0,0 +1,203 @@
+Shape,Description,Common usecase,Depend on,Note,SVG
```

Comment:
@vishalvivekm, @SAHU-1 in order to maintain this data file long-term, I propose converting to YAML.